recipes/napari-picasso/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "napari-picasso" %}
+{% set version = "0.0.0" %}
```

Comment:
Probably the wrong version :)

Reply:
https://pypi.org/project/napari-PICASSO/

<img width="461" alt="Screen Shot 2022-06-10 at 10 21 58 AM" src="https://user-images.githubusercontent.com/3627835/173098034-013e5d91-17db-4ebf-a8a5-89c873fc1c55.png">

It is very very very alpha @krpandit  ?

Reply:
yea I forgot to update that line.

It's pretty alpha, there are few more features I need to add, but the very basics should work.

Reply:
See https://github.com/nygctech/PICASSO/issues/1

Reply:
This has been fixed :)

Thanks @krpandit !